contracts/src/v0.1/mocks/RequestResponseConsumerMock.sol

```diff
@@ -25,12 +38,11 @@ contract RequestResponseConsumerMock is RequestResponseConsumerBase {
     // Receive remaining payment from requestDataPayment
     receive() external payable {}
 
-    function requestData(
+    function requestDataInternal(
```

Comment:
@KelvinThai I still think both the `requestDataInternal` and `requestDataDirectPaymentInternal` do not make sense. Each request asks for different data type, but use the same request definition (fetching KLAY-USD price from cryptocompare.com). Can we make it better?

Reply:
If we do not plan to create a  separate request for each data type, we should at least explain it for future readers.